crates/stdarch-gen/neon.spec

```diff
@@ -601,6 +601,15 @@ generate float64x*_t
 arm = vmul.
 generate float*_t
 
+/// Divide
+name = vdiv
+fn = simd_div
+a = 2.0, 6.0, 4.0, 10.0
+b = 1.0, 2.0, 1.0, 2.0
+validate 2.0, 3.0, 4.0, 5.0
+
+aarch64 = fdiv
```

Comment:
Can you generate arm versions of this which use the `vdiv` instruction?

Reply:
Note that arm only supports `f32` variants: `f64` is aarch64-only.

Reply:
According to the docs the supported architectures are only A64: https://developer.arm.com/architectures/instruction-sets/simd-isas/neon/intrinsics?search=vdiv

> Supported architectures
> A64

Maybe the docs are wrong?

Reply:
Ah you are correct, it seems the ARM `vdiv` instruction only supports scalar division, not vector division.